src/utils/presenter.ts

```diff
@@ -13,6 +15,15 @@ export const getPresenterPage = (req: Request, res: Response, next: NextFunction
     const appData: ApplicationData = getApplicationData(req.session);
     const presenter = appData[PresenterKey];
 
+    if (isRemoveJourney(req)){
+      return res.render(templateName, {
+        journey: config.JourneyType.remove,
+        backLinkUrl: `${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
```

Comment:
The back link on this page is showing as http://chs.local/update-an-overseas-entity/update-filing-date?journey=remove, which is different to ${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}
not sure what has happened but the page is not using the value you have passed in for backLinkUrl

Reply:
i think it might be related to the git merge I had to do to get this page working in docker. When you do your merge, check that the values passed in to the template are correct.

Reply:
This one is my error, i hadn't updated to the latest version of this branch. I'll mark this one as resolved but there is still an issue with the backlink when you get a validation error, that one is mentioned in a different comment about the navigation.ts.